modules/admin/src/resources/js/directives.js

```diff
@@ -659,6 +659,7 @@
             scope: {
                 "model": "=",
                 "options": "=",
+                "dateFormat": "=",
```

Comment:
Sorry about my mistake! 

I will change validate

```
   			var res = n.split(".");
            			if (res.length == 3) {
            				if (res[2].length == 4) {
            					var en = res[1] + "/" + res[0] + "/" + res[2];
		        				$scope.model = (Date.parse(en)/1000);
		        				$scope.datePickerToggler = false;
            				}
}
```
to

```
$scope.model = (Date.parse(n)/1000);
$scope.datePickerToggler = false;
```

Are you think it ok?